android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/MeasureReportViewModel.kt

```diff
@@ -360,6 +389,27 @@ constructor(
       }
   }
 
+  /** Returns a Map of year-month list for for all months falling in given measure period */
```

Comment:
can you make this a proper kdoc string please? https://kotlinlang.org/docs/kotlin-doc.html#kdoc-syntax